schema/mysql-migrations/upgrade_188.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE director_activity_log
+DROP INDEX checksum,
+ADD UNIQUE INDEX checksum (checksum);
+
+ALTER TABLE director_generated_config
```

Comment:
The constraint should be removed beforehand. Otherwise this will lead to duplicate key errors.

Reply:
Done